src/services/ProfileApiService.js

```diff
@@ -37,7 +37,16 @@ export async function patchProfile(username, params) {
         'Content-Type': 'application/merge-patch+json',
       },
     },
-  );
+  ).catch((error) => {
+    const processedError = Object.create(error);
+    const fieldErrors = Object.entries(processAccountData(error.response.data.field_errors))
```

Comment:
Is it possible for there to be more than one error per field?

Reply:
(Since it was an array, it seems like it's at least plausible...)

Reply:
It doesn't appear that we get arrays back. I tried by making two errors in social links and I only get the first